NEWS.md

```diff
@@ -20,7 +23,7 @@
   (#1563, @krlmlr).
 
 * `@importFrom` works again for quoted non-syntactic names, e.g.
-  `@importFrom magrittr "%>%"` or ``@importFrom rlang `:=` `` 
+  `@importFrom magrittr "%>%"` or ``@importFrom rlang `:=` ``
```

Comment:
Looks like you accidentally stripped trailing whitespace.

Reply:
Thanks Positron! :/